mod_reforged/hooks/skills/actives/reload_bolt.nut

```diff
@@ -5,19 +5,19 @@
 		create();
 		this.m.IsRemovedAfterBattle = false;
 	}
-	// Overwrite vanilla function to remove the line which removes this skill upon being used
-	// Instead we set IsHidden to true and in shoot_bolt and shoot_stake we set its IsHidden to false
+
+	// Overwrite vanilla function to prevent removal of this skill
 	o.onUse = function( _user, _targetTile	)
 	{
 		this.consumeAmmo();
 		this.getItem().setLoaded(true);
-		this.m.IsHidden = true;
 		return true;
 	}
 
-	function onCombatFinished()
+	o.isHidden <- function()
 	{
-		this.skill.onCombatFinished();
-		this.m.IsHidden = true;
+		if (!this.getContainer().getActor().isPlacedOnMap()) return true;	// In the character screen this is always hidden
```

Comment:
This might make it always hidden in weapon tooltip (because the dummy player is never placed on map). Can you check if the skill shows up in weapon tooltips?

Reply:
![image](https://user-images.githubusercontent.com/2252464/226938694-dedaa1db-860f-49a9-8ae0-3b8c376238f8.png)

Looks to be shown just fine and I don't know how that's possible from how you explain the dummy working. 
But I also didn't make the Dummy or looked into it

Reply:
Ah that makes sense. Hidden skills _do_ show up in the weapon tooltip, so my original comment was based on a confusion. This is fine.